deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
I hate to do this, but I think we may need to reopen this conversation... Do we want to have another migration type script to make sure that data that matches the previous index pattern matches the new one so that users will be able to see their data in Kibana still?  We would update this to grab anything that doesn't start with the prefix and create an alias for it that matches `$prefix.$project.$uuid.$somesuffix` or something.